<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter</class>
 <widget class="QDialog" name="Parameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>119</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущие значения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>97</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дисбаланс траверсы с шатунами, кг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_vd">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_vd">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_vd">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>310</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Радиус вращения НГШ, мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_r">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_r">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_r">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Фи, град</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_fi">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fi">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_fi">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>131</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Уравновешивающий момент, Н*м</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_m_ur">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_m_ur">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_m_ur">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>63</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Номинальное тяговое усилие станка-качалки, кН</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_f">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_f">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_f">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string> Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>240</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Длина шатуна, мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_p">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_p">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>169</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Переднее плечо, мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_a">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_a">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_a">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>203</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Заднее плечо, мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_c">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_c">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_c">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>274</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Расстояние между опорой балансира и валом ред., мм</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ui_k">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_k">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_k">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
